frontend/src/scalars/ui/Chart.vue

```diff
@@ -0,0 +1,598 @@
+<template>
+    <div class="visual-dl-page-charts">
+        <div class="visual-dl-chart-box" :style="computedStyle">
+        </div>
+        <div class="visual-dl-chart-actions">
+            <v-btn @click="expandArea">
+                <v-icon size="20">settings_overscan</v-icon>
+            </v-btn>
+            <v-select
+                v-if="downloadLink"
+                style="width:100px;"
```

Comment:
If you have a fixed value, why not putting it in css

Reply:
coz of laziness